README.md

```diff
@@ -8,10 +8,22 @@
 This package provides basic interaction with a Clojure subprocess (REPL).
 It's based on ideas from the popular `inferior-lisp` package.
 
-`inf-clojure` has two components - a nice Clojure REPL with
-auto-completion and a minor mode (`inf-clojure-minor-mode`), which
-extends `clojure-mode` with commands to evaluate forms directly in the
-REPL.
+`inf-clojure` aims to expose the extensive self-documenting features
+of Clojure repls into an emacs package. Inf-clojure does not require
+middleware or complicated machinery, but it more a predefined set of
+commands for a repl. Because of this, you can interact with many more
+repl types than an nREPL-based tool like CIDER. Inf-clojure supports
+the following repls:
+- Clojure
+- clojurescript through cljs.main (no piggieback required)
+- planck
+- lumo
+- joker
+- bb
```

Comment:
I'd spell out Babashka here, as `bb` probably means nothing to people who haven't used it.